pyuploadtool/changelog/changelog.py

```diff
@@ -0,0 +1,106 @@
+
+import os
+
+
+# order them by priority
+# https://cheatography.com/albelop/cheat-sheets/conventional-commits/
+CONVENTIONAL_COMMIT_SPEC = {
+    "feat":     "Features",
+    "fix":      "Bug Fixes",
+    "perf":     "Performance Improvements",
+    "docs":     "Documentation",
+    "ci":       "Continuous Integration",
+    "refactor": "Refactoring",
+    "test":     "Tests",
+    "build":    "Builds",
+    "revert":   "Reverts",
+    "chore":    "Chores",
+    "others":   "Commits"
+}
+
+
+class Changelog:
+    def __init__(self, title=os.getenv("CHANGELOG_TITLE"), commit_link_prefix=None):
```

Comment:
In Python, one should _never_ have some default value other than `None`. There's more than one reason for this. Also, I don't see where this environment variable is coming from. It's also not documented below.

It'll be easier to explain this if you implement your fallback with a `if ... is None` check, as it gives you more space for explanations.

Reply:
ok, will fix right away